src/main/java/uk/gov/pay/ledger/payout/entity/PayoutEntity.java

```diff
@@ -1,17 +1,36 @@
 package uk.gov.pay.ledger.payout.entity;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import uk.gov.pay.ledger.event.model.serializer.MicrosecondPrecisionDateTimeDeserializer;
+import uk.gov.pay.ledger.event.model.serializer.MicrosecondPrecisionDateTimeSerializer;
+
 import java.time.ZonedDateTime;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
 public class PayoutEntity {
 
-    private final Long id;
-    private final String gatewayPayoutId;
-    private final Long amount;
-    private final ZonedDateTime createdDate;
-    private final ZonedDateTime paidOutDate;
-    private final String statementDescriptor;
-    private final String status;
-    private final String type;
+    @JsonIgnore
+    private Long id;
+    private String gatewayPayoutId;
+    private Long amount;
+    @JsonIgnore
+    private ZonedDateTime createdDate;
+    @JsonSerialize(using = MicrosecondPrecisionDateTimeSerializer.class)
+    @JsonDeserialize(using = MicrosecondPrecisionDateTimeDeserializer.class)
+    private ZonedDateTime paidOutDate;
+    private String statementDescriptor;
```

Comment:
Think this field can go into payout_details json blob ?

Reply:
payout details doesn't yet exist in this PR, I think because the payout will rarely if ever change in size it's okay to have something like this at the top -- we can definitely move it once payout_details is in if that's proposed